spyder/config/main.py

```diff
@@ -424,7 +425,7 @@
               'editor/move line up': "Alt+Up",
               'editor/move line down': "Alt+Down",
               'editor/go to new line': "Ctrl+Shift+Return",
-              'editor/go to definition': "Ctrl+G",
+              'editor/go to definition': "F3" if MAC else "Ctrl+G",
```

Comment:
Perhaps we should change this shortcut to `Ctrl+J`? I mean, I've heard that it's hard to access `Fn` keys in Mac, so it could be worth considering this too.

What do you think?

Reply:
Ok, I don't have a strong opinion either, so let's leave it like this and see what users think about it.